--- branches/KDE/4.3/kdesdk/kate/app/kateviewdocumentproxymodel.cpp	2009/06/24 10:17:04	986143
+++ branches/KDE/4.3/kdesdk/kate/app/kateviewdocumentproxymodel.cpp	2010/01/20 23:32:20	1077835
@@ -207,7 +207,13 @@
 
   m_current = index;
   m_markOpenedTimer->start(100);
-  //sort();
+
+  // This to protect Multiple call to sort()
+  // As this function is getting called on KateFileList clicked event to open and activate doc
+  // So if the sender is KateFileList ignore sorting.
+  KateFileList* f = qobject_cast<KateFileList*>(sender());
+  if(!f)
+    sort();
 }
 
 void KateViewDocumentProxyModel::slotMarkOpenedTimer()
@@ -566,8 +572,6 @@
     m_mapToSource[m_mapFromSource[i]] = i;
   }
   endInsertRows();
-  if (m_sortRole == Qt::DisplayRole || m_sortRole == KateDocManager::UrlRole)
-    sort();
 }
 void KateViewDocumentProxyModel::slotRowsRemoved ( const QModelIndex & parent, int start, int end )
 {
